src/main/java/com/skytask/controller/ProductController.java

```diff
@@ -0,0 +1,39 @@
+package com.skytask.controller;
+
+import com.skytask.model.Product;
+import com.skytask.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+
+@Controller
+public class ProductController {
+
+    private ProductService productService;
+
+    @Autowired
```

Comment:
Nitpick: you don't need @Autowired to inject and Object. since 1.4 spring version.